src/test/resources/ack/jms-ack.xml

```diff
@@ -14,7 +14,8 @@
     </flow>
 
     <flow name="listener" initialState="started">
-        <jms:listener config-ref="config" destination="${destination}" ackMode="${ack.mode}" numberOfConsumers="1"/>
+        <jms:listener config-ref="config" destination="${destination}"
```

Comment:
not that I found, no. Actually I've added one in the previous PR when async was discussed